src/components/common/Tree/Tree.js

```diff
@@ -30,17 +105,22 @@ const DynamicTreeView = ({
   // types based on TreeView types
   const onToggle = (_event, nodeIds) => setExpandedItems(nodeIds);
 
+  // show only folder items in the navigation tree
+  const itemsFiltered = items.filter(item => item.type === "folder");
```

Comment:
Use `ITEM_TYPES` instead of the string `folder`.

Reply:
Solved